### ISSUE TITLE
Bot suddently crashes for no reason

### ISSUE THREAD

**Issue by egils-consulting** [jhsoby/telegram-wikilinksbot#15]

**Comment by egils-consulting** [comment]
I suggest adding the logging module and debug it that way, we are interested in what is going on right before the crash so we can reproduce and find the bug.

**Comment by jhsoby** [comment]
When it crashes, the only hint (thus far) in the command line is "Killed.". Googling leads me to believe that this is because it uses too much memory for some reason – I've tried to limit what it uses already with garbage collection, which worked quite well (before I did, this would happen several times a week – now it can be more than a month between each time it happens).

I don't actually need a perfect solution – closing out the program and restarting it doesn't affect anything, so perhaps a simple cron job to do that once a day would do the trick?
